newscoop/themes/system_templates/editor_image.tpl

```diff
@@ -1,6 +1,6 @@
 {{ dynamic }}
 	{{ if $imageDetails['align'] }} <div align="center"> {{ /if }}
-		<div class="cs_img {{ if $imageDetails['align'] }}cs_fl_{{ $imageDetails['align'] }}{{ /if }}" style="{{ if $imageDetails['width'] }}width:{{ $imageDetails['width'] }};{{ /if }}">
+		<div class="cs_img {{ if $imageDetails['align'] }}cs_fl_{{ $imageDetails['align'] }}{{ /if }}" {{ if $imageDetails['percentage'] }}style="width:{{ $imageDetails['percentage'] }};"{{ /if }}>
```

Comment:
its not needed, because the size in pixels will be passed directly to the get_img route as a parameter.